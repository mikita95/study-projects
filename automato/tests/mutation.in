5 3
11111
00000
11110
00000
10110
00111