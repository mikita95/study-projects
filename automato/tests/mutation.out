0.00032
0.00128
0.02048
